src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1444,10 +1441,18 @@ h3.variant {
 	height: 100%;
 	display: block;
 	background-color: var(--button-background-color);
+	border: 1px solid var(--border-color);
+	border-radius: 2px;
 }
 
 #copy-path {
 	color: var(--copy-path-button-color);
+	background: var(--main-background-color);
+	height: 34px;
+	margin-left: 10px;
+	padding: 0;
```

Comment:
Since you removed the `padding: 5px` from `#settings-menu > a, #help-button > a, #copy-path`, I think you can remove this one too since there is no `padding` to override.

Reply:
Not anymore thanks to normalize:

![image](https://user-images.githubusercontent.com/3050060/196275373-a054afde-6bd7-4dfe-970d-53ee50c10155.png)

Reply:
Oh sorry, you're right. My bad.